src/main/java/org/folio/qm/service/population/LeaderMarcPopulationService.java

```diff
@@ -18,11 +17,13 @@ public void populate(QuickMarc qmRecord) {
       return;
     }
 
-    var leader = populateValues(initialLeader, new LinkedList<>(COMMON_CONSTANT_LEADER_ITEMS));
+    var leader = populateValues(initialLeader, new LinkedList<>(getConstantLeaderItems()));
```

Comment:
```suggestion
    var leader = populateValues(initialLeader, getConstantLeaderItems());
```
I think there is no need for additional list creation.